webv2/updateTableSchema/addColumn.go

```diff
@@ -0,0 +1,34 @@
+package updateTableSchema
```

Comment:
updateTableSchema seems like an action , better to rename it to something more meaningful and generic , maybe table

Reply:
Renamed to **table**